src/index.html

```diff
@@ -4,10 +4,56 @@
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <title>Stars</title>
+    <title>stars__stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars--0 container">
```

Comment:
fix it everywhere, you'll understand it later when you'll learn BEM
```suggestion
    <div class="stars stars--0">
```